core/modes/mode_select.talon

```diff
@@ -0,0 +1,4 @@
+win.title: /VIM MODE:s/
+-
+
+tag(): user.vim_select_mode
```

Comment:
this should be `vim_mode_select`

Reply:
we agreed to remove the select mode at the end, since we don't see any use for now